[BUG]If #define DEBUG_VIA_WEB is selected, a compilation error occurs.

**Describe the bug*
If DEBUG_VIA_WEB is selected several compilation errors occurs.

Software/src/charger/CHEVY-VOLT-CHARGER.cpp:74:67: error: 'struct CAN_frame' has no member named 'MsgID'
   74 |       logging.printf("CAN Rcv unknown frame MsgID=%x\n", rx_frame.MsgID);

Software/src/charger/CHEVY-VOLT-CHARGER.cpp:152:75: error: 'charger_stat_ACvol' was not declared in this scope
  152 |     logging.printf("Charger AC in IAC=%fA VAC=%fV\n", charger_stat_ACcur, charger_stat_ACvol);

Software/src/charger/CHEVY-VOLT-CHARGER.cpp:153:56: error: 'charger_stat_HVcur' was not declared in this scope
  153 |     logging.printf("Charger HV out IDC=%fA VDC=%fV\n", charger_stat_HVcur, charger_stat_HVvol);
Software/src/charger/CHEVY-VOLT-CHARGER.cpp:154:56: error: 'charger_stat_LVcur' was not declared in this scope

Software/src/charger/CHEVY-VOLT-CHARGER.cpp:154:56: error: 'charger_stat_LVcur' was not declared in this scope
  154 |     logging.printf("Charger LV out IDC=%fA VDC=%fV\n", charger_stat_LVcur, charger_stat_LVvol);

**To Reproduce**
Please try to explain the steps required to reproduce the issue
Add or Remove the \\ back before DEBUG_VIA_WEB

**Expected behavior**
A clear and concise description of what you expected to happen.
No errors from compiler

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Version and settings (please complete the following information):**
- Please attach your USER_SETTINGS files for easier debugging
- Software version: 12. (latest from may 12 2025)
- Battery used: [FAKE_BATTERY)]
- Inverter communication protocol: [BYD_MODBUS]
- Hardware used for Battery-Emulator: [HW_LILYGO]
- CONTACTOR_CONTROL: [yes]
- MQTT: [yes]

Possible solution.
Correct the CHEVY-VOLT-CHARGER.cpp according the error messages.

--- a/Software/src/charger/CHEVY-VOLT-CHARGER.cpp
+++ b/Software/src/charger/CHEVY-VOLT-CHARGER.cpp
@@ -71,7 +71,7 @@ void ChevyVoltCharger::map_can_frame_to_variable(CAN_frame rx_frame) {
       break;
     default:
 #ifdef DEBUG_LOG
-      logging.printf("CAN Rcv unknown frame MsgID=%x\n", rx_frame.MsgID);
+      logging.printf("CAN Rcv unknown frame MsgID=%x\n", rx_frame.ID);
 #endif
       break;
   }
@@ -149,9 +149,9 @@ void ChevyVoltCharger::transmit_can(unsigned long currentMillis) {
   /* Serial echo every 5s of charger stats */
   if (currentMillis - previousMillis5000ms >= INTERVAL_5_S) {
     previousMillis5000ms = currentMillis;
-    logging.printf("Charger AC in IAC=%fA VAC=%fV\n", charger_stat_ACcur, charger_stat_ACvol);
-    logging.printf("Charger HV out IDC=%fA VDC=%fV\n", charger_stat_HVcur, charger_stat_HVvol);
-    logging.printf("Charger LV out IDC=%fA VDC=%fV\n", charger_stat_LVcur, charger_stat_LVvol);
+    logging.printf("Charger AC in IAC=%fA VAC=%fV\n", AC_input_current(), AC_input_voltage());
+    logging.printf("Charger HV out IDC=%fA VDC=%fV\n", HVDC_output_current(), HVDC_output_voltage());
+    logging.printf("Charger LV out IDC=%fA VDC=%fV\n", LVDC_output_current(), LVDC_output_voltage());
     logging.printf("Charger mode=%s\n", (charger_mode > MODE_DISABLED) ? "Enabled" : "Disabled");
     logging.printf("Charger HVset=%uV,%uA finishCurrent=%uA\n", setpoint_HV_VDC, setpoint_HV_IDC, setpoint_HV_IDC_END);
   }

